components/Buttons/AddButton.js

```diff
@@ -0,0 +1,24 @@
+import StyledButton from "./StyledButton";
+import SVGIcon from "../Icons/SVGIcon";
+
+export default function AddButton({ isAddMode, onClick }) {
+  return (
+    <>
```

Comment:
This fragment is only needed if you want to return two or more elements in parallel. Here you only return the StyledButton so you can remove the fragment.

Reply:
done!